src/main/resources/interface-specification.yml

```diff
@@ -319,6 +319,34 @@ paths:
             application/problem+json:
               schema:
                 $ref: '#/components/schemas/Problem'
+  '/bulk/calcAgreementsState/consumers/{consumerId}/attributes/{attributeId}':
```

Comment:
I'm not sure about the naming of this route - I would use bulk when the request payload is a bunch of something to send in one request.
For example, we use the verb load for load attributes.
Maybe here we could use calculate or recalculate.
`/compute/agreementsState/consumers/{consumerId}/attributes/{attributeId}`
or
`/recalculate/agreementsState/consumers/{consumerId}/attributes/{attributeId}`

Reply:
makes sense 👍 
I'll change it to `compute`